lib/tic_tac_toe.ex

```diff
@@ -1,28 +1,31 @@
 defmodule TicTacToe do
   alias TicTacToe.Display
-  alias TicTacToe.Console
-  alias TicTacToe.Presenter
   alias TicTacToe.Player
-  alias TicTacToe.HumanPlayer
   alias TicTacToe.Board
 
-  def run(opts \\ []) do
-    board = Keyword.get(opts, :board, List.duplicate(nil, 9))
-    display = Keyword.get(opts, :display, %Console{})
-    presenter = Keyword.get(opts, :presenter, Presenter)
-    player = Keyword.get(opts, :player, %HumanPlayer{token: "X"})
-
-    new_display = %{display | state: show_board(board, presenter, display)}
-    play_turn(player, board) |> show_board(presenter, new_display)
+  def run(state) do
+    state
+    |> show_board()
+    |> play_turn()
+    |> show_board()
+    |> play_turn()
+    |> show_board()
+    |> Map.get(:display)
```

Comment:
Do we need to return the display? Or can we return the `state`?

Reply:
We can return state.  I'll adjust.